bazel/osx/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"] + ["**/*"]),
+)
+
+exports_files([
+    "xcode_locator.m",
+])
+
+DARWIN_XCODE_LOCATOR_COMPILE_COMMAND = """
+  /usr/bin/xcrun clang -fobjc-arc -framework CoreServices \
+      -framework Foundation -o $@ $<
+"""
+
+genrule(
```

Comment:
I was able to remove most of the copied bazel files. I've left the `configure_osx_toolchain` invocation in `cc_configure.bzl`.